chainer/dataset/tabular/tabular_dataset.py

```diff
@@ -147,6 +147,30 @@ def fetch(self):
         elif self.mode is None:
             return examples[0]
 
+    def convert(self, data):
+        """Convert fetched data.
+
+        This method takes data fetched by :meth:`fetch` and
+        pre-process them before passing them to models.
+        The default behaviour is converting each column into an ndarray.
+        This behaviour can be overrided by :meth:`with_converter`.
```

Comment:
```suggestion
        This behaviour can be overridden by :meth:`with_converter`.
```
Sorry, it's my bad :(